pyproject.toml

```diff
@@ -32,6 +32,7 @@ dev = [
     "mypy==1.15.*",
     "pre-commit==4.2.*",
     "pytest==8.3.*",
+    "pytest-asyncio>=1.1.0",
```

Comment:
fixing this down to patch version will prevent us from semi automatic updates to the lates patched version. Everywhere else we peg only major and minor version. SO this would be better `==1.1.*`